### ISSUE TITLE
md-datepicker not working with aurelia-cli bundle

### ISSUE THREAD

**Issue by avrahamcool** [aurelia-ui-toolkits/aurelia-materialize-bridge#379]
I created a project with aurelia-cli 0.24.0 (ESnext)
I'm using the latest `aurelia-materialize-bridge` (0.24.0)
and the latest `materialize-css` (0.98.0)

I followed the [install/bundle instruction](https://aurelia-ui-toolkits.gitbooks.io/materialize-bridge-docs/content/installation/aurelia-cli.html) from the tutorial to the letter.

I'm using the `useAll()` helper function to load all of the bridge components, and its working fine excepts when I try to use `md-datepicker`.

I'm trying the code from the [basic use example](http://aurelia-ui-toolkits.github.io/demo-materialize/#/samples/datepicker-basic-use) and I get the error

```
Unhandled rejection TypeError: $(...).pickadate is not a function
    at MdDatePicker.bind (http://localhost:9000/scripts/materialize-bundle.js:21055:37)
    at Controller.bind (http://localhost:9000/scripts/vendor-bundle.js:23200:24)
    at View.bind (http://localhost:9000/scripts/vendor-bundle.js:21204:24)
    at Controller.bind (http://localhost:9000/scripts/vendor-bundle.js:23192:19)
    at Controller.automate (http://localhost:9000/scripts/vendor-bundle.js:23137:12)
    at http://localhost:9000/scripts/vendor-bundle.js:24224:20
From previous event:
    at CompositionEngine._createControllerAndSwap (http://localhost:9000/scripts/vendor-bundle.js:24223:45)
    at CompositionEngine.compose (http://localhost:9000/scripts/vendor-bundle.js:24309:21)
    at TemplatingEngine.compose (http://localhost:9000/scripts/vendor-bundle.js:24571:38)
    at Aurelia.setRoot (http://localhost:9000/scripts/vendor-bundle.js:13466:21)
    at http://localhost:9000/scripts/app-bundle.js:78:22
From previous event:
    at Object.configure (http://localhost:9000/scripts/app-bundle.js:77:21)
    at http://localhost:9000/scripts/vendor-bundle.js:12464:22
From previous event:
    at config (http://localhost:9000/scripts/vendor-bundle.js:12459:48)
    at handleApp (http://localhost:9000/scripts/vendor-bundle.js:12450:12)
    at http://localhost:9000/scripts/vendor-bundle.js:12483:13
From previous event:
    at http://localhost:9000/scripts/vendor-bundle.js:12481:40
From previous event:
    at http://localhost:9000/scripts/vendor-bundle.js:12480:29
From previous event:
    at run (http://localhost:9000/scripts/vendor-bundle.js:12476:26)
    at Object.<anonymous> (http://localhost:9000/scripts/vendor-bundle.js:12503:3)
    at Object.execCb (http://localhost:9000/scripts/vendor-bundle.js:5894:33)
    at Module.check (http://localhost:9000/scripts/vendor-bundle.js:5081:51)
    at Module.enable (http://localhost:9000/scripts/vendor-bundle.js:5374:22)
    at Object.enable (http://localhost:9000/scripts/vendor-bundle.js:5755:39)
    at Module.<anonymous> (http://localhost:9000/scripts/vendor-bundle.js:5359:33)
    at http://localhost:9000/scripts/vendor-bundle.js:4332:23
    at each (http://localhost:9000/scripts/vendor-bundle.js:4257:31)
    at Module.enable (http://localhost:9000/scripts/vendor-bundle.js:5311:17)
    at Module.init (http://localhost:9000/scripts/vendor-bundle.js:4986:26)
    at http://localhost:9000/scripts/vendor-bundle.js:5658:36
printWarning @ vendor-bundle.js:1395
```

apperantly, the `picker`  module is not loaded.

**Comment by Thanood** [comment]
Strange.. I just tried it with a JS app. Are you using TypeScript by any chance? There have been some problems when using TS.. I can upload my app to a repo for comparison.

**Comment by avrahamcool** [comment]
No, I'm using ESnext. 
I'd like to see your working code.

**Comment by Thanood** [comment]
Here it is: https://github.com/Thanood/md-bridge-cli

**Comment by avrahamcool** [comment]
your project is working indeed.
i'll close this issue and check my code again.

**Comment by Thanood** [comment]
Okay, great.
If you don't mind, I'd be interested in what causes your issue. 😃

**Comment by avrahamcool** [comment]
I'll check it again tomorrow first thing in the morning.
I hope I'll find the cause.

**Comment by avrahamcool** [comment]
the problem was I am also using other libraries like `aurelia-kendoui-bridge` who also depends on `jquery`.
so my `vendor-bundle` includes `jquery` as a dependency.
as I said, I followed the tutorial and created a new bundle for `aurelia-materialize-bridge` who also included a copy of `jquery` (that copy is enhanced with `picker` functionality).
but at run time, when `aurelia-materialize-bridge` required an instance of `jquery`, it resolved to the one from `vendor-bundle`.

I fixed it by removing the inclusion of `jquery` in the `aurelia-materialize-bridge` bundle.
now I have one `jquery` inclusion in my `vendor` bundle to serve all the libraries who need it.
